packages/flutter_timeline_view/lib/src/screens/timeline_post_overview_screen.dart

```diff
@@ -0,0 +1,46 @@
+// ignore_for_file: prefer_expression_function_bodies
+
+import 'package:flutter/material.dart';
+import 'package:flutter_timeline_interface/flutter_timeline_interface.dart';
+import 'package:flutter_timeline_view/flutter_timeline_view.dart';
+
+class TimelinePostOverviewScreen extends StatelessWidget {
+  const TimelinePostOverviewScreen({
+    required this.timelinePost,
+    required this.options,
+    super.key,
+  });
+  final TimelinePost timelinePost;
+  final TimelineOptions options;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(options.translations.postOverview),
+      ),
+      body: Column(
+        children: [
+          Flexible(
+            child: TimelinePostScreen(
+              userId: timelinePost.creatorId,
+              options: options,
+              post: timelinePost,
+              onPostDelete: () {},
+              service: TimelineService(postService: LocalTimelinePostService()),
```

Comment:
you should get the service from the options or as an argument. I think the service is also used for fetching things like images